### ISSUE TITLE
[UX Review] Message bundle should be editable

### ISSUE THREAD

**Issue by xianli123** [keycloak/keycloak-admin-ui#1915]
### Describe the bug

In the new design, users can edit the message bundle as shown in the prototype here https://marvelapp.com/prototype/56d9606/screen/76458141. Thanks!

<img width="1085" alt="Screen Shot 2022-01-21 at 3 56 47 PM" src="https://user-images.githubusercontent.com/53931519/150491683-c4fc6a89-9312-491e-bc40-493ccf85d3a9.png">

### Version

New Admin UI

**Comment by ssilvert** [comment]
@xianli123 I don't understand this one.  It does look like it's only allowing you to edit English.  Is that what you mean?

**Comment by xianli123** [comment]
@ssilvert Sorry. I put the wrong screenshot above.

**Comment by xianli123** [comment]
I mean that users should be allowed to edit the message bundle table based on the Supported locales. And I found this issue has been fixed in the latest implementation as shown in the screenshot below. Therefore, we can close this issue.

<img width="1094" alt="Screen Shot 2022-02-07 at 2 44 10 PM" src="https://user-images.githubusercontent.com/53931519/152738231-80f34e67-026d-4736-9909-22e4334af57e.png">
